roles/chromium/templates/chromium-latest

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+{{playbook_dir}}/chromium-latest-linux/run.sh $*
```

Comment:
@jugglinmike This only works if you've run the `update.sh` at least once. That is, unless you run either `update-and-run` or `update` first, `run.sh` doesn't do anything. I know you changed this so it doesn't update every time, but maybe there should be an initial call to update in the ansible script?